test/playlist.test.js

```diff
@@ -385,11 +385,119 @@ function(assert) {
 
   assert.equal(seekable.start(0), 0, 'starts at the earliest available segment');
   assert.equal(seekable.end(0),
-              9 - (2 + 2 + 1),
-              'allows seeking no further than three segments from the end');
+               // Playlist duration is 9s. Target duration 2s. Seekable end should be at
+               // least 6s from end. Adding segment durations starting from the end to get
+               // that 6s target
+               9 - (2 + 2 + 1 + 2),
+               'allows seeking no further than three times target duration from the end');
```

Comment:
If we're going by this comment, shouldn't it be `9 - 3 * 2`?

Reply:
So I tried to make this more clear by including the wording `Adding segment durations starting from the end to get that 6s target`. Basically the rule we are following is that we cannot start playback any later than `lastSegment Duration + 2 * targetDuration` from the end of the playlist (which reminds me I need to update this comment as its still saying `3 * targetDuration`). However, for robustness sake, instead of just setting seekable end to be `PlaylistEnd - (lastSegment Duration + 2 * targetDuration)` We find the segment that that value falls under and use that segment's start time as seekable end. So in this case, the safest point we could start loading from is at 3 seconds. We have a segment that goes from 2-4s, so we set seekable end to the start of that segment, which is 2s

Reply:
does that make sense? any ideas on how i can make that more clear in the comment?

Reply:
Maybe `allows seeking no further than the start of the segment 2 target durations back from the beginning of the last segment`?